scripts/qemu-wrapper

```diff
@@ -310,31 +335,25 @@ def main(argv):
        qemu_env["LD_PRELOAD"] = "/usr/lib64/libjemalloc.so.1:" + qemu_env["LD_PRELOAD"]
     qemu_env["MALLOC_CONF"] = "narenas:1,tcache:false,lg_dirty_mult:22"
 
-    core_dump_limit = enable_core_dumps()
-    f.write("core dump limit: %d\n" % core_dump_limit)
+    sys.stdout.flush()
+    sys.stderr.flush()
 
-    pid = os.getpid()
-    xenstore_write("/local/domain/%d/qemu-pid" % domid, "%d" % pid)
+    qemu = subprocess.Popen(qemu_args, executable=qemu_dm, env=qemu_env,
+                            stdout=subprocess.PIPE, stderr=subprocess.STDOUT,
```

Comment:
Quite good and smart change!
A minor comment here: It is better to judge whether we should redirect the stderr of subprocess to stdout based on whether fd 1 and 2 is same.
In this case, https://github.com/xapi-project/xenopsd/blob/master/xc/device.ml#L1867 explicit indicates redirecting stderr to stdout, however someday we change it, we have to change these two places.

Reply:
The logger process only has a single `stdin` so it is required to connect both `stdout` and `stderr` to the single `stdin` as I did. What do you expect to happen if fd 1 is different from fd 2?

Reply:
If `fd1` is different from `fd2`, we just need to redirect the output of `fd1` to `fd0` of logger, that means the value of `let redirect_stderr_to_stdout = true` in https://github.com/xapi-project/xenopsd/blob/master/xc/device.ml#L1863 is false.